src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,151 @@
+    }
+
+    @Test
+    void loginIsNull_NotOk() {
+        user.setLogin(null);
+        Assertions.assertThrows(ImpossibleRegisterUserException.class,
+                ()-> {
+            registrationService.register(user);
+                });
+    }
+
+    @Test
+    void loginNotExist_Ok() {
+        user.setLogin("Deadpool");
+        try {
+            Assert.assertEquals(user, registrationService.register(user));
+        } catch (ImpossibleRegisterUserException e) {
+            System.out.println("Can`t register this user");
+        }
+    }
+
+    @Test
+    void loginHaveSizeLessThanAllowable_NotOk() {
+        user.setLogin("sanya");
+        Assertions.assertThrows(ImpossibleRegisterUserException.class,
+                ()-> {
+            registrationService.register(user);
+                });
+    }
+
+    @Test
+    void loginHaveSizeEqualsAllowableSize_Ok() {
+        user.setLogin("sanya2");
+        try {
+            Assert.assertEquals(user, registrationService.register(user));
+        } catch (ImpossibleRegisterUserException e) {
+            System.out.println("Can`t register this user");
+        }
+    }
+
+    @Test
+    void loginIsBiggerThanAllowableSize_Ok() {
+        user.setLogin("sanya1256");
+        try {
+            Assert.assertEquals(user, registrationService.register(user));
+        } catch (ImpossibleRegisterUserException e) {
+            System.out.println("Can`t register this user");
+        }
+    }
+
+    @Test
+    void ageOfUserLessThanAllowableAge() {
+        user.setAge(15);
+        Assertions.assertThrows(ImpossibleRegisterUserException.class,
+                ()-> {
+            registrationService.register(user);
+                });
+    }
+
+    @Test
+    void ageOfUserEqualsAllowableAge_Ok() {
+        user.setAge(18);
+        try {
+            Assert.assertEquals(user, registrationService.register(user));
+        } catch (ImpossibleRegisterUserException e) {
+            System.out.println("Can`t register this user");
+        }
+    }
+
+    @Test
+    void ageOfUserBiggerThanAllowableAge_Ok() {
+        user.setAge(45);
+        try {
+            Assert.assertEquals(user, registrationService.register(user));
+        } catch (ImpossibleRegisterUserException e) {
+            System.out.println("Can`t register this user");
+        }
+    }
+
+    @Test
+    void passwordLengthOfUserLessThanAllowableLength_NotOk() {
+        user.setPassword("gyfyt");
+        Assertions.assertThrows(ImpossibleRegisterUserException.class,
+                ()->{
+            registrationService.register(user);
+                });
+    }
+
+    @Test
+    void passwordLengthOfUserEqualsAllowableLength_Ok() {
+        user.setPassword("121123");
+        try {
+            Assert.assertEquals(user, registrationService.register(user));
+        } catch (ImpossibleRegisterUserException e) {
+            System.out.println("Can`t register this user");
+        }
+    }
+
+    @Test
```

Comment:
The test method name `passwordLengthOfUserBiggerThanAllowableLength_Ok` does not follow the required naming convention `<methodUnderTest>_<state>_<expectedBehavior>`. Consider renaming it to something like `register_PasswordExceedsMinimumLength_Success`.